linked-lists/lib/linkedList.js

```diff
@@ -80,24 +81,62 @@ class LinkedList {
   }
 
 
-  // serialize is O(1)
-  serialize() {
-    return JSON.stringify(this);
-  }
+  // // serialize is O(1)
+  // serialize() {
+  //   return JSON.stringify(this);
+  // }
 
   //deserialize is O(n)
-  deserialize() {
-    this.head = listItem.head;
-    this.tail = listItem.tail;
-    this.length = listItem.length;
-
-    //QUESTION: i think i need to use static, why is it saying it is reserved. Static methods are used when we need a function bound to a class, but not to any object of that class. 
-    static deserialize(listObject) {
-    let newList = JSON.parse(listObject);
-    return new LinkedList(newList);
+  // deserialize() {
+  //   this.head = listItem.head;
+  //   this.tail = listItem.tail;
+  //   this.length = listItem.length;
+
+  //   //QUESTION: i think i need to use static, why is it saying it is reserved. Static methods are used when we need a function bound to a class, but not to any object of that class. 
+  //   // static 
```

Comment:
Any keyword like `class`, `function`, `static`, etc. is a `reserved` keyword.
In order to use it, the syntax would look like this:
```
static deserialize(){
  // code goes here
}
```